### ISSUE TITLE
[vue/max-attributes-per-line] Attribute "class" should be on a new line for <a></a> tag

### ISSUE THREAD

**Issue by dmitry-saritasa** [vuejs/eslint-plugin-vue#376]
**Tell us about your environment**

* **ESLint Version:** 4.17.0
* **eslint-plugin-vue Version:**: 4.2.2
* **Node Version:** 8.9.4

**Please show your full configuration:**

```
// https://eslint.org/docs/user-guide/configuring

module.exports = {
  root: true,
  parserOptions: {
    parser: 'babel-eslint'
  },
  env: {
    browser: true,
    es6: true,
    jquery: true,
  },
  // https://github.com/vuejs/eslint-plugin-vue#priority-a-essential-error-prevention
  // consider switching to `plugin:vue/strongly-recommended` or `plugin:vue/recommended` for stricter rules.
  // 'plugin:vue/essential',

  extends: [
    'plugin:vue/essential',
    'plugin:vue/recommended',
    'plugin:vue/strongly-recommended',
    'airbnb-base',
    'prettier',
  ],
  // required to lint *.vue files
  plugins: [
    'vue',
    'prettier',
    'eslint-comments',
    'compat',
  ],
  // check if imports actually resolve
  settings: {
    'import/resolver': {
      webpack: {
        config: 'build/webpack.base.conf.js'
      }
    }
  },
  // add your custom rules here
  rules: {
    // don't require .vue extension when importing
    'import/extensions': ['error', 'always', {
      js: 'never',
      vue: 'never'
    }],
    // disallow reassignment of function parameters
    // disallow parameter object manipulation except for specific exclusions
    'no-param-reassign': ['error', {
      props: true,
      ignorePropertyModificationsFor: [
        'state', // for vuex state
        'acc', // for reduce accumulators
        'e' // for e.returnvalue
      ]
    }],
    // allow optionalDependencies
    'import/no-extraneous-dependencies': ['error', {
      optionalDependencies: ['test/unit/index.js']
    }],
    'import/newline-after-import': 'error',
    'no-plusplus': 'off',
    'max-params': ['error', 3],
    'comma-style': ['error', 'last'],
    'comma-dangle': ['error', 'always'],
    'require-jsdoc': ['error', {
      'require': {
        'FunctionDeclaration': true,
        'MethodDefinition': true,
        'ClassDeclaration': true,
        'ArrowFunctionExpression': true,
        'FunctionExpression': true
      }
    }],
    'padding-line-between-statements': [
      'error',
      {
        'blankLine': 'always',
        'prev': ['const', 'let', 'var'],
        'next': '*'
      },
      {
        'blankLine': 'always',
        'prev': '*',
        'next': ['const', 'let', 'var']
      },
      {
        'blankLine': 'any',
        'prev': ['const', 'let', 'var'],
        'next': ['const', 'let', 'var']
      },
      {
        'blankLine': 'always',
        'prev': '*',
        'next': ['case', 'default']
      },
      {
        'blankLine': 'always',
        'prev': ['const', 'let', 'var', 'block', 'block-like'],
        'next': ['block', 'block-like']
      },
      {
        'blankLine': 'always',
        'prev': 'directive',
        'next': '*'
      },
      {
        'blankLine': 'any',
        'prev': 'directive',
        'next': 'directive'
      },
      {
        'blankLine': 'always',
        'prev': '*',
        'next': 'return'
      }
    ],
    'no-bitwise': ['error', { 'int32Hint': true }],
    'eslint-comments/disable-enable-pair': 'error',
    'eslint-comments/no-duplicate-disable': 'error',
    'eslint-comments/no-unlimited-disable': 'error',
    'eslint-comments/no-unused-disable': 'warn',
    'eslint-comments/no-unused-enable': 'warn',
    // browser compatibility error
    // Use caniuse and @kangax's compat table for determining coverage
    'compat/compat': 'error',
    // allow debugger during development
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    // vuejs
    'vue/max-attributes-per-line': [2,
      {
        'singleline': 1,
        'multiline': {
          'max': 1,
          'allowFirstLine': false
        }
      }
    ],
  }
};

```

**What did you do? Please include the actual source code causing the issue.**

```vue

<div class="col-md-4 slideshow">
            <img src="@/assets/product4.jpg" >
            <a class="arrow-btn"
               href="#">Manage</a>
            <div id="chart">
              <sparkline label="Views" :id="id+'clicks'"/>
            </div>
          </div>



```

**What did you expect to happen?**

class="arrow-btn" is not expected to be on the next line, due to 
```
    'vue/max-attributes-per-line': [2,
      {
        'singleline': 1,
        'multiline': {
          'max': 1,
          'allowFirstLine': false
        }
      }
    ],
```
**What actually happened? Please include the actual, raw output from ESLint.**

```
eslint --cache 'src/**/**/*.js' 'src/**/**/*.vue'
/home/dmitry/Projects/frontend/src/components/house/Product.vue
  43:16  error  Attribute "class" should be on a new line                                  vue/max-attributes-per-line
  46:40  error  Attribute "bind" should be on a new line                                   vue/max-attributes-per-line
  71:5   error  Prop "id" should define at least its type                                  vue/require-prop-types
  75:3   error  The "components" property should be above the "props" property on line 70  vue/order-in-components
```

**Comment by dmitry-saritasa** [comment]
solved with 'allowFirstLine': true
